modules/video/clearAllClips.py

```diff
@@ -4,7 +4,7 @@
 
 
 
-def main():
+def main(object):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'object'](https://www.codacy.com/app/saoron_3361/Cardigan/file/2711783945/issues/source?bid=3323288&fileBranchId=3340798#l7)